application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -13,6 +13,7 @@
 import org.togetherjava.tjbot.features.MessageReceiverAdapter;
 
 import java.awt.Color;
+import java.util.concurrent.ScheduledFuture;
```

Comment:
where is that import coming from? looks unecessary

Reply:
Its Extra I will remove it